days/04-06-collections/chris/movies_by_year.py

```diff
@@ -0,0 +1,26 @@
+# This little project will show you a sorted by rating
+# list of movies that was released in a given year
+
+from collections import defaultdict, namedtuple, Counter
+import csv
+
+Movie = namedtuple('Movie','title, rating')
+
+years_of_movies = defaultdict(list)
+
+with open('movie_metadata.csv', encoding='utf-8') as f:
+    movies_csv = csv.DictReader(f)
+    for row in movies_csv:
+        movie = Movie(title=row['movie_title'],rating=row['imdb_score'])
+        years_of_movies[row['title_year']].append(movie)
+    
+def get_movies_by_year(year):
+    print(f'Here are all the movies released in {year} sorted by rating:',end='\n\n')
```

Comment:
Getting the below error originating from line 18.  Could be me, could we take a look today before I merge this?

`Bryan-Tobin's-MacBook-Pro:folder bryantobin$ python test.py
  File "test.py", line 18
    print('Here are all the movies released in {year} sorted by rating:',end='\n\n')
                                                                            ^
SyntaxError: invalid syntax`

Reply:
It looks like your print is missing the "f" that would precede the first quotation mark after the opening parentheses. What version of Python are you using?